documentation/rails.md

```diff
@@ -0,0 +1,124 @@
+Rails configuration
+===================
+
+To use this file with Rails you have two choices:
+
+#### 1. Adding a new `webpack_asset_url` helper
+
+You'll need a helper to replace asset pipeline. Here is an example how you an do this:
+
+```Ruby
+module WebpackHelper
+  def webpack_asset_url(asset)
+    "/assets/#{manifest.fetch(asset)}"
+  end
+
+  def manifest
+    @manifest ||= JSON.parse(File.read('manifest.json'))
+  rescue
+    fail 'Please run webpack'
+  end
+end
+```
+
+This file would then be saved in `app/helpers/webpack_helper.rb` for example.
+
+Now you can use it like this:
+
+```HTML
+<img src="#{webpack_asset_url('logo.svg')}" alt="logo" />
+```
+
+to require your assets. [Here](https://github.com/infinum/webpack-rails-manifest-plugin/blob/master/example/webpack.config.js#L12) is an example on how to add a digest to a file.
+
+**Take note that you can't use Rails `image_tag`, `stylesheet_link_tag`, `javascript_include_tag` helper.**
+
+#### 2. Monkey patching rails assets helpers
+
+Be sure to remove rails sprockets entirely by editing your `applicaton.rb` file
+
+```Ruby
+# Comment this out
+# require 'rails/all'
+
+# Include every module separately
+require 'rails'
+# Pick the frameworks you want:
+require "active_model/railtie"
+require "active_job/railtie"
+require "active_record/railtie"
+require "action_controller/railtie"
+require "action_mailer/railtie"
+require "action_view/railtie"
+# require "sprockets/railtie"
+require "rails/test_unit/railtie"
+```
+
+And after that copy the code below to your initializers folder `app/config/initializers/webpack-rails-manifest.rb`
```

Comment:
Is that And in the beginning really necessary? Maybe just rephrase the whole thing a bit, this is also lacking punctuation. Maybe:

"Then save the following code in the initializers directory as `app/config/initializers/webpack-rails-manifest.rb`."